{"name": "test_div_int_param[10-0-None]", "status": "broken", "statusDetails": {"message": "ZeroDivisionError: division by zero", "trace": "number1 = 10, number2 = 0, exception = None\n\n    @pytest.mark.happy\n    @pytest.mark.parametrize(\"number1,number2,exception\", {\n        (10, 2, 5),\n        (10, 5, 2),\n        (100000000, 1, 100000000),\n        (100, 200, 0.5),\n        (10, 3, 3.33),\n        (10.2, 0.2, 51),\n        (10, 'a', 'b'),\n        ('abc', 10, 'd'),\n        (10, 0, None),\n        (0, 10, 0)\n    })\n    def test_div_int_param(number1, number2, exception):\n>       assert div(number1, number2) == exception\n\nunit\\test_div.py:73: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\na = 10, b = 0\n\n    def div(a, b):\n        #todo:健壮性、异常处理\n>       return a/b\nE       ZeroDivisionError: division by zero\n\nunit\\div.py:4: ZeroDivisionError"}, "parameters": [{"name": "number1", "value": "10"}, {"name": "number2", "value": "0"}, {"name": "exception", "value": "None"}], "start": 1583118100602, "stop": 1583118100602, "uuid": "27f2812e-aba1-4f93-a451-30a006e443e7", "historyId": "36baaf1e5b32896ff54ec1c6a77ffe65", "testCaseId": "11c1c6507a9ba2abc0386ed88a58468f", "fullName": "unit.test_div#test_div_int_param", "labels": [{"name": "tag", "value": "happy"}, {"name": "parentSuite", "value": "unit"}, {"name": "suite", "value": "test_div"}, {"name": "host", "value": "DESKTOP-2P3JG1L"}, {"name": "thread", "value": "6744-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "unit.test_div"}]}